README.md

```diff
@@ -60,34 +60,37 @@ a simple, free locking mechanism, and enforcing best practices around CLI usage
 
 ## Quick start
 
-Go into the folder with your Terraform templates and create a `.terragrunt` file. This file uses the same
-[HCL](https://github.com/hashicorp/hcl) syntax as Terraform. Here is an example `.terragrunt` file that configures
+Go into the folder with your Terraform configurations and create a `terraform.tfvars` file. Terraform reads this file 
+by default for [variables](https://www.terraform.io/intro/getting-started/variables.html#from-a-file); Terragrunt reads
+this file by default for its configuration. Here is an example `terraform.tfvars` file that configures
 Terragrunt to use [DynamoDB for locking](#locking-using-dynamodb) and to [automatically manage remote
 state](#managing-remote-state) for you using the [S3 backend](https://www.terraform.io/docs/state/remote/s3.html):
 
 ```hcl
-# Configure Terragrunt to use DynamoDB for locking
-lock = {
-  backend = "dynamodb"
-  config {
-    state_file_id = "my-app"
+terragrunt = {
+  # Configure Terragrunt to use DynamoDB for locking
+  lock {
+    backend = "dynamodb"
+    config {
+      state_file_id = "my-app"
+    }
   }
-}
-
-# Configure Terragrunt to automatically store tfstate files in an S3 bucket
-remote_state = {
-  backend = "s3"
-  config {
-    encrypt = "true"
-    bucket = "my-bucket"
-    key = "terraform.tfstate"
-    region = "us-east-1"
+  
+  # Configure Terragrunt to automatically store tfstate files in an S3 bucket
+  remote_state {
+    backend = "s3"
+    config {
+      encrypt = "true"
+      bucket = "my-bucket"
+      key = "terraform.tfstate"
+      region = "us-east-1"
+    }
   }
 }
 ```
```

Comment:
> What happens when Terraform itself encounters these directives in a terraform.tfvars? Does it just ignore them?

Yup, it just ignores them.

> I believe we previously implemented various interpolation functionality. Do test validate that still works as expected?

Yup, the automated tests check a lot of interpolation functionality in `.tfvars` files.

> We should still explicitly clarify the above points.

Good idea. Added: https://github.com/gruntwork-io/terragrunt/pull/122/commits/c550d8f47243ee4854bd0c1edce5470d1df6e772